package.json

```diff
@@ -31,7 +31,7 @@
   },
   "homepage": "https://github.com/ankeetmaini/react-infinite-scroll-component#readme",
   "peerDependencies": {
-    "react": ">=0.14.0",
+    "react": ">=16.9.0",
```

Comment:
`getDerivedStateFromProps` is introduced in [16.3.0](https://github.com/facebook/react/blob/master/CHANGELOG.md#react-dom-17)
```suggestion
    "react": ">=16.3.0",
```

Reply:
Sorry for this little mistake. My bad.
Btw, thanks for pointing that out.